src/LibHdb++MySQL.cpp

```diff
@@ -556,7 +641,7 @@ void HdbPPMySQL::cache_err_id(string error_desc, int &ERR_ID)
 #ifdef _LIB_DEBUG
 	cout << __func__<< ": entering for '"<<error_desc << "' map size=" << attr_ERR_queue.size() << endl;
 #endif
-	map<string,int>::iterator it = attr_ERR_ID_map.find(error_desc);
+	unordered_map<string,int>::iterator it = attr_ERR_ID_map.find(error_desc);
```

Comment:
What's your stand on using auto ? I know I would use it in this case, I find iterator type too long to not use auto, but I am lazy…

Reply:
It's simply habit. I agree iterators are too long to write, but at the same time code could be more readable seeing the type